src/Widgets/EligibilityModal/modal.test.tsx

```diff
@@ -108,4 +108,21 @@ describe('Modal', () => {
       expect(element).toHaveTextContent('150,00 €')
     })
   })
+  describe('empty plans', () => {
```

Comment:
We should add a test for the ModalContainer

Reply:
done 👍